<launch>

     <node name="optical_flow_estimator" pkg="iarc7_sensors"
     type="iarc7_flow_transformer">
      	<rosparam command="load"
      	file="$(find iarc7_sensors)/param/flow_transformer.yaml" />

     </node>

</launch>
